### ISSUE TITLE
[FEATURE REQUEST] Fixed time install

### ISSUE THREAD

**Issue by gabriel-vanca** [Devolutions/UniGetUI#1294]
### Please confirm these before moving forward

- [X] I have searched for my feature proposal and not found a work-in-progress/duplicate/resolved/discarded issue.
- [X] This proposal is a completely new feature. If you want to suggest an improvement or an enhancement, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=enhancement&projects=&template=enhancement-improvement.yml&title=%5BENHANCEMENT%5D+%28Enter+your+description+here%29)

### Describe the new feature

Rather than the current "check every few hours", it would be great if I would also have the option to:
- install updates at a certain time of the day (e.g. 2am, when no one uses the PC)
- install updates only at certain times of the week (e.g. install only in weekends)

It would be great if I would still get the notifications about an update being available in the same way as soon (e.g. every 1 hour) so that I have the option to update an app earlier than the set time if I so decide.

Please add command line options for these settings too.

### Describe how this new feature could help users.

I don't want my productivity disrupted by lots of update updates and UAC prompts in the middle of the working day.

**Comment by marticliment** [comment]
> I don't want my productivity disrupted by lots of update updates and UAC prompts in the middle of the working day

You can enable the auto-elevate feature

**Comment by marticliment** [comment]
> Would be great if that would go away too.

This prompt is impossible to remove unless you run WingetUI as administrator by default.

**Comment by mamoreau-devolutions** [comment]
I agree the UAC prompts can be annoying, but they can't be skipped for obvious reasons. I'm closing for now, but we have plans to see what could be done for enterprise-wide deployments with an elevated agent to perform the installations alongside UniGetUI, without the UAC prompt.
